docs/xtrabackup_bin/restoring_individual_tables.md

```diff
@@ -12,61 +12,55 @@ a int(11) DEFAULT NULL
 
 ## Export the table
 
-Created the table in innodb_file_per_table mode, so
-after taking a backup as usual with the –backup option, the
-.ibd file exists in the target directory:
+To generate an .ibd file in the target directory, create the table using the `innodb_file_per_table` mode:
 
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
 /data/backups/mysql/test/export_test.ibd
 ```
 
-when you prepare the backup, add the –export option to the
-command. Here is an example:
+During the `--prepare` step, add the `--export` option to the
+command. For example:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/data/backups/mysql/
 ```
 
-!!! note
-   
-    If you restore an encrypted InnoDB tablespace table, add the  keyring file:
+When restoring an encrypted InnoDB tablespace table, add the keyring file:
 
-    ```{.bash data-prompt="$"}
-    $ xtrabackup --prepare --export --target-dir=/tmp/table \
-    --keyring-file-data=/var/lib/mysql-keyring/keyring
-    ```
+```{.bash data-prompt="$"}
+$ xtrabackup --prepare --export --target-dir=/tmp/table \
+--keyring-file-data=/var/lib/mysql-keyring/keyring
+```
 
-Now you should see an .exp file in the target directory:
+The following files are the only files required to import the table into a server running Percona Server for MySQL with XtraDB or MySQL 8.0. If the server uses InnoDB Tablespace Encryption, add the .cfp file, which contains the transfer key and an encrypted tablespace key.
 
-```{.bash data-prompt="$"}
-$ find /data/backups/mysql/ -name export_test.*
-/data/backups/mysql/test/export_test.exp
+The files are located in the target directory:
+
+```text
 /data/backups/mysql/test/export_test.ibd
 /data/backups/mysql/test/export_test.cfg
 ```
 
-These three files are the only files required to import the table into a server running
-*Percona Server for MySQL* with *XtraDB* or *MySQL* 8.0. In case the server uses InnoDB,
-Tablespace Encryption adds an additional .cfp file which contains the transfer key and an encrypted tablespace key.
+## Import the table
 
-!!! note
-   
-    The .cfg metadata file contains an *InnoDB* dictionary dump in a special format. This format is different from the .exp one which is
-    used in *XtraDB* for the same purpose. A .cfg\` file is not required to import a tablespace to *MySQL* 8.0 or Percona
-    Server for MySQL 8.0.
+On the destination server running Percona Server for MySQL with XtraDB or MySQL 8.0, create a table with the same structure, and then perform the following steps:
 
-    A tablespace is imported successfully even if the table is from
-    another server, but *InnoDB* performs a schema validation if the corresponding .cfg file is located in the same directory.
+1. Run the `ALTER TABLE test.export_test DISCARD TABLESPACE;` command. If you see the following error message:
 
-## Import the table
+    ??? example "Error message"
```

Comment:
Not sure if there is a problem in my browser, I see three "?" here

Reply:
We use ??? example "Error message" to add expandable tabs. Just text formatting.

Reply:
Ah ok. Thanks for the clarification.